src/components/Card.jsx

```diff
@@ -61,7 +61,7 @@ export const CardExecMember = ({ admin }) => {
   const fullName = admin.fullName || 'DEFAULT'
   const linkedIn = admin.linkedIn || 'DEFAULT'
   const position = admin.position || 'DEFAULT'
-  const img = admin.profilePicture.fluid || ''
+  const img = (admin.profilePicture && admin.profilePicture.fluid) || ''
```

Comment:
Would it make more sense to just do `admin.profilePicture?.fluid` to handle the null checks?

Reply:
I wasn't sure what version this repo deploys to on CodePipeline (there's no files on it in the repo afaik and there's no `.nvm` file), and this syntax was introduced in Node 14, so I used this syntax. If you think it's worth checking out the CodePipeline deployment let me know

Reply:
Good observation, let's leave it then